src/mono/mono/mini/mini-llvm.c

```diff
@@ -4700,6 +4700,24 @@ emit_llvmonly_handler_start (EmitContext *ctx, MonoBasicBlock *bb, LLVMBasicBloc
 		mono_llvm_emit_clear_exception_call (ctx, ctx->builder);
 	}
 
+#ifdef TARGET_WASM
```

Comment:
Is this going to be called only for catch clauses?  will it pop twice if there's a catch and a finally?

Reply:
Its not really a pop, it just gets rid of all lmf frames below the current one, so its ok to call it multiple times.